src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect } from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Typography from '@material-ui/core/Typography';
+import { profileStyles } from './Profile.styles';
+import {
+  Avatar,
+  FormControl,
+  Grid,
+  InputLabel,
+  OutlinedInput,
+  DialogTitle,
+  DialogActions,
+  Container,
+} from '@material-ui/core';
+import { useSelector } from 'react-redux';
+import { ReduxState } from '../../redux/combinedReducer';
+
+interface ProfileProps {
+  open: boolean;
+  handleClose: () => void;
+}
+
+const Profile = (props: ProfileProps) => {
+  const user = useSelector((state: ReduxState) => {
+    return state.user;
+  });
+  const classes = profileStyles();
+
+  const [name, setName] = React.useState(user.name);
+  const [email, setEmail] = React.useState(user.email);
+  const [phone, setPhone] = React.useState(user.phone);
+
+  useEffect(() => {
+    setName(user.name);
+    setEmail(user.email);
+    setPhone(user.phone);
+  }, [user.name, user.email, user.phone]);
+
+  const nameInput = React.createRef();
+  const emailInput = React.createRef();
+  const phoneInput = React.createRef();
+
+  const handleNameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setName(event.target.value);
+  };
+
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+
+  const handlePhoneChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPhone(event.target.value);
+  };
+
+  function handleSaveChanges() {
+    // Read values
+    // @ts-ignore
+    console.log(nameInput.current.value);
+    // @ts-ignore
+    console.log(emailInput.current.value);
+    // @ts-ignore
+    console.log(phoneInput.current.value);
+
+    // Check if input changed, if not, do props.handleClose()
+    // Validate input
+    // Send to API to update Firebase
+    props.handleClose();
+  }
+
+  return (
```

Comment:
Remove all the aria-labels and ids, makes it harder to read and you don't need it

Reply:
Done